.pre-commit-config.yaml

```diff
@@ -107,5 +107,5 @@ repos:
         name: markdownlint
         entry: markdownlint --fix --ignore grafana-plugin/node_modules --ignore grafana-plugin/dist --ignore docs **/*.md
       - id: markdownlint
-        name: markdownlint - docs
-        entry: markdownlint --fix -c ./docs/.markdownlint.json ./docs/**/*.md
+        name: markdownlint - docs/sources
+        entry: markdownlint --fix --ignore README.md -c ./docs/.markdownlint.json ./docs/sources/**/*.md
```

Comment:
I don't think `README.md` should have to be ignore'd here. `./docs/sources/**/*.md` is telling `markdownlint` to only consider files in that directory

Reply:
Yeah it's weird but without this additional change, `markdownlint` was linting `README.md` and finding errors. I didn't want to spend any more time working out why so I just ignored the file specifically.

Reply:
👍